runners/spark/src/test/java/org/apache/beam/runners/spark/translation/streaming/ResumeFromCheckpointStreamingTest.java

```diff
@@ -65,10 +65,7 @@
 /**
  * Tests DStream recovery from checkpoint - recreate the job and continue (from checkpoint).
  *
- * <p>Tests Aggregators, which rely on Accumulators - Aggregators should be available, though
- * state is not preserved (Spark issue), so they start from initial value.
- * //TODO: after the runner supports recovering the state of Aggregators, update this test's
- * expected values for the recovered (second) run.
+ * <p>Tests Aggregators, which rely on Accumulators - Aggregators should be available.
```

Comment:
Perhaps this could be phrased a bit clearer in terms of the goals of this test?
Something along the lines of `Tests that aggregator values are resorted upon recovery from checkpoint`.